webapp/src/ts/services/xml-forms.service.ts

```diff
@@ -264,11 +268,14 @@ export class XmlFormsService {
           // fallback for backwards compatibility
           return this.getByView(internalId);
         }
+        console.error(`Error in XMLFormService getByView : the form "${internalId}" was not found`);
         throw err;
       })
       .then(doc => {
         if (!this.findXFormAttachmentName(doc)) {
-          return Promise.reject(new Error(`The form "${internalId}" doesn't have an xform attachment`));
+          const errorMsg = `The form "${internalId}" doesn't have an xform attachment`;
+          console.log('Error in XMLFormService getByView : '+errorMsg);
```

Comment:
I believe this is supposed to be a `console.error`? Also, the error is in `XMLFormService.get`. 
```suggestion
          console.log('Error in XMLFormService.get: ' + errorMsg);
```